Extensions/retags.js

```diff
@@ -119,16 +122,17 @@ XKit.extensions.retags = {
 				$target = $element.find('.notification-wrapper');
 				url = $target.find('a').not('.notification-username').attr('href');
 			}
-			if (url) {
+			if (url && !host && !id) {
```

Comment:
is this just for clarification or did you run into a bug here? I can't see the path where both url and host would be set.